wp-menu-cart.php

```diff
@@ -12,6 +12,14 @@
  * WC tested up to: 5.1.0
  */
 
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+if ( class_exists( 'WpMenuCart' ) || class_exists( 'WPO_Menu_Cart_Pro' ) ) {
```

Comment:
Maybe it's a good idea to add a notice like "You already have WooCommerce Menu Cart / WP Menu Cart installed. This plugin has the same features, so you can delete it" or something like that, what do you think?

Reply:
It's more of a safety measure. The chance of this happening is very unlikely to start with. The point here is to avoid loading the plugin class if a copy is already loaded under a different name. If we put a notice we're already venturing into duplication territory (although closures could avoid this).